resources/lang/en/community.php

```diff
@@ -126,25 +126,4 @@
             'instructions' => 'click the heart button to proceed to the osu!store',
         ],
     ],
-    'slack' => [
```

Comment:
Some of these translations are still in use, so we probably don't want to remove them all...